index.d.ts

```diff
@@ -84,3 +84,23 @@ const ab: Merge<Foo, Bar> = {a: 1, b: 2};
 ```
 */
 export type Merge<FirstType, SecondType> = Omit<FirstType, Extract<keyof FirstType, keyof SecondType>> & SecondType;
+		  
+/**
+ * Diffs two objects.
+ *
+ * Given objects with types T and V, returns an object that has all the keys in T that do not also exist in V.
+ *
+ * @example `type Safe = Diff<AllProperties, UnsafeProperties>`
```

Comment:
Can you include a more comprehensive example? Should also include the import statement.